proposals/build-cli-options.md

```diff
@@ -0,0 +1,50 @@
+# [Proposal] Allow additional cli options to pass to the build command
+
+Related to:
+- https://github.com/devcontainers/spec/discussions/301
+- https://github.com/devcontainers/spec/pull/324
+- https://github.com/microsoft/vscode-remote-release/issues/3545
+
+## Goal
+
+Extend the build section of the configuration with an option to pass additional arguments via cli.
+
+## Proposal
+
+Introduce the property `cliOptions` to the `buildOptions` in the schema. It is very similar to `runArgs`, which contains cli-arguments passed to the `run` command, the `cliOptions` passes cli-arguments to the `build` command.
```

Comment:
I wonder if calling the new property as `options` makes sense. (eg `build.options`)

// cc @bamurtaugh @craiglpeters @Chuxel for product specific insights. Also, there are many other helpful suggestions from the author in the https://github.com/devcontainers/spec/pull/324 description. 

> build.buildArgs
build.cliArgs
build.cliOptions
build.options
build.commandArgs
build.commandOptions

Reply:
I am open for the naming. `options` seems a bit too generic for me. In the end, the field represents cli arguments passed to the command.

Reply:
Thanks for providing all the great suggestions! I lean toward `build.options` too - it feels the cleanest and most concise. While other areas may also have things called "options," I don't anticipate us wanting to refer to other things as `build.options`.

Reply:
I renamed it to `options`.